pallenec

```diff
@@ -63,15 +64,42 @@ local inspect_opts = {
     end
 }
 
+local function print_tokens(lexer)
+    while true do
+        local token = lexer:next()
+        if token.name == "EOF" then
+            break
+        end
+
+        local loc = token.loc
+        local end_loc = token.end_loc
+        local value = ""
+        if token.value then
+            local s = token.value
+            if type(token.value) == "string" then
+                s = C.string(token.value)
+            end
+            value = string.format(" %s", s)
+        end
+        print(string.format("%d:%d,%d:%d,%d:%d %s%s", loc.line, loc.col,
+            end_loc.line, end_loc.col, loc.pos, end_loc.pos, token.name, value))
```

Comment:
It isn't super clear that the first two `%d:%d` are line and column, but the third `%d:%d` are absolute positions.  Can you think of a way to make it clearer that they are different things?